src/components/organisms/EarthEditor/CanvasArea/convert.ts

```diff
@@ -166,14 +177,64 @@ export const convertWidgets = (data: GetEarthWidgetsQuery | undefined): Widget[]
     .map(
       (widget): Widget => ({
         id: widget.id,
+        extended: widget.extended as boolean,
```

Comment:
```suggestion
        extended: !!widget.extended,
```

You don't need to do down casting.